src/Equinor.ProCoSys.IPO.WebApi/Synchronization/BusReceiverService.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.ServiceBus;
+using System.Text.Json;
+using Equinor.ProCoSys.IPO.Domain;
+using Equinor.ProCoSys.IPO.Domain.AggregateModels.InvitationAggregate;
+using Equinor.ProCoSys.BusReceiver;
+using Equinor.ProCoSys.BusReceiver.Interfaces;
+using Equinor.ProCoSys.BusReceiver.Topics;
+
+namespace Equinor.ProCoSys.IPO.WebApi.Synchronization
+{
+    public class BusReceiverService : IBusReceiverService
+    {
+        private readonly IInvitationRepository _invitationRepository;
+        private readonly IPlantSetter _plantSetter;
+        private readonly IUnitOfWork _unitOfWork;
+
+        public BusReceiverService(IInvitationRepository invitationRepository, IPlantSetter plantSetter, IUnitOfWork unitOfWork)

```

Comment:
To be discussed:
Should the updates be handled by using Commands and MediatR? (I.e: no repos and unitofwork here)

Reply:
I see the point, but we must make sure the sync is not compared to "normal updates", meaning not called by other controllers etc. I think we are best served with an "isolated" update here. But we can reconsider when we extend to Preservation/PO